riff-cli/cmd/init.go

```diff
@@ -160,18 +159,13 @@ func InitGo(initOptions *options.InitOptions) (*cobra.Command, *options.InitOpti
 		Long:  utils.InitGoCmdLong(),
 
 		RunE: func(cmd *cobra.Command, args []string) error {
-			initOptions.Handler,_ = cmd.Flags().GetString("handler")
 			if initOptions.Handler == "" {
-				initOptions.Handler = initOptions.FunctionName
+				initOptions.Handler = strings.Title(initOptions.FunctionName)
 			}
-			err := initializers.Go().Initialize(*initOptions)
-			if err != nil {
-				return err
-			}
-			return nil
+			return initializers.Go().Initialize(*initOptions)
 		},
 	}
-	initGoCmd.Flags().String("handler", "", "the name of the function handler")
+	initGoCmd.Flags().StringVar(&initOptions.Handler, "handler", "", "the name of the function handler (Exported go symbol)")
```

Comment:
What does "Exported go symbol" mean here?

Reply:
That the handler should be the name of the Exported go symbol in the go plugin

Reply:
Maybe change to "Exported go function name"?

Reply:
or "name of Exported go function"?

Reply:
yeah, that sounds better - can change on merge